app/views/locations/ips/new.html.erb

```diff
@@ -10,13 +10,20 @@
 
     <%= form_with model: @ip, url: location_ips_path do |form| %>
       <div class="govuk-form-group <%= field_error(@ip, :address) %>">
-        <%= form.label :ip, "Enter IP address (IPv4 only)", class: "govuk-label" %>
-        <%= form.text_field :address, id: :address, autocomplete: "off", class: "govuk-input" %>
-      </div>
+        <%= form.label :address, "Enter IP address (IPv4 only)", class: "govuk-label" %>
+        <%= form.text_field :address, id: 'address', autocomplete: "off", class: "govuk-input", multiple: true %>
```

Comment:
`multiple: true` worked pretty nicely for the POST here - makes it so the addresses get picked up as an array in params, in the controller.